File: Step5-Python/2562.py
#way1
arr=[0]*9

for i in range(9):
    arr[i]=int(input())
    
max_num=max(arr)
print(max_num)
print(arr.index(max_num)+1)

#way2
arr=[]

for i in range(9):
    arr.append(int(input()))
    
print(max(arr))
print(arr.index(max(arr))+1)

File: Step5-C/2562.c
#define _CRT_SECURE_NO_WARNINGS
#include <stdio.h>

int main()
{
	int arr[9];
	
	for (int i = 0; i < 9; i++)
		scanf("%d", &arr[i]);

	int max_num = arr[0];
	int index = 0;
	for (int j = 0; j < 9; j++)
	{
		if (arr[j] > max_num)
		{
			max_num = arr[j];
			index = j;
		}
	}
	printf("%d\n%d", max_num, index + 1);
	return 0;
}